ambry-cloud/src/test/java/com.github.ambry.cloud/azure/MockChangeFeedQuery.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud.azure;
+
+import com.github.ambry.cloud.CloudBlobMetadata;
+import com.microsoft.azure.cosmosdb.rx.AsyncDocumentClient;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.Mockito.*;
+
+
+public class MockChangeFeedQuery extends CosmosDataAccessor {
+  private final Map<String, String> continuationTokenToBlobIdMap = new HashMap<>();
+  private final Map<String, String> blobIdToContinuationTokenMap = new HashMap<>();
+  private final Map<String, CloudBlobMetadata> blobIdToMetadataMap = new HashMap<>();
+  private int continuationTokenCounter = -1;
+
+  MockChangeFeedQuery() {
+    super(mock(AsyncDocumentClient.class), "", mock(AzureMetrics.class));
+  }
+
+  /**
+   * Add a blobid to the change feed.
+   * @param cloudBlobMetadata {@link CloudBlobMetadata} to add.
+   */
+  void add(CloudBlobMetadata cloudBlobMetadata) {
+    String blobId = cloudBlobMetadata.getId();
+    blobIdToMetadataMap.put(blobId, cloudBlobMetadata);
+    if (blobIdToContinuationTokenMap.containsKey(blobId)) {
+      continuationTokenToBlobIdMap.put(blobIdToContinuationTokenMap.get(blobId), null);
+    }
+    continuationTokenToBlobIdMap.put(Integer.toString(++continuationTokenCounter), blobId);
+    blobIdToContinuationTokenMap.put(blobId, Integer.toString(continuationTokenCounter));
+  }
+
+  public String queryChangeFeed(String requestContinuationToken, int maxFeedSize, List<CloudBlobMetadata> changeFeed,
```

Comment:
Because of this override, CosmosDataAccessor.queryChangeFeed() has no test coverage.

Reply:
created tests for CosmosDataAccessor.queryChangeFeed() in CosmosDataAccessorTest.java